### PR TITLE
Link back to rbac from use-cases

### PR DESCRIPTION
**Pull request by emmanuella194**
Worked on the documentation page  (jupyterhub/docs/source/rbac/use-case.md)

Added a wikipedia reference to [RBCA framework] and also emphasized on the solution under the *service to cull idle servers*

### CODE REVIEW

File: docs/source/rbac/use-cases.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ To determine which scopes a role should have, one can follow these steps:
 5. Customize the scopes with filters if needed
 6. Define the role with required scopes and assign to users/services/groups/tokens
 
-Below, different use cases are presented on how to use the RBAC framework.
+Below, different use cases are presented on how to use the [RBAC framework](https://en.wikipedia.org/wiki/Role-based_access_control).
```
[the remark this answers is not in the document]
Comment (emmanuella194) [reply docs/source/rbac/use-cases.md]: Thanks for the swift feedback, I have added a fix, kindly review

File: docs/source/rbac/use-cases.md [hunk 2]
```diff
@@ -9,12 +9,12 @@ To determine which scopes a role should have, one can follow these steps:
 5. Customize the scopes with filters if needed
 6. Define the role with required scopes and assign to users/services/groups/tokens
 
-Below, different use cases are presented on how to use the RBAC framework.
+Below, different use cases are presented on how to use the [RBAC framework](https://jupyterhub.readthedocs.io/en/stable/rbac/index.html).
```
Comment (minrk) [inline docs/source/rbac/use-cases.md:12]: Sorry, it should be an _internal_ link. I believe you can use `[RBAC framework](./index.md)`

Comment (emmanuella194) [reply docs/source/rbac/use-cases.md:12]: Thank you for the correction, I will make the necessary changes

**Comment by minrk** [comment]
Thank you, @emmanuella194!

**Comment by emmanuella194** [comment]
Thank you minrk for your support

**Pull request merged**